Project.toml

```diff
@@ -23,7 +23,7 @@ UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
 [compat]
 DiffEqBase = "6.47"
 DiffEqCallbacks = "2.14"
-EllipsisNotation = "0.4"
+EllipsisNotation = "0.4, 1.0"
```

Comment:
Shouldn't we just go all-in and make 1.0 the minimum?

Reply:
fd3c4c87229ff3388adbda3686918b95665e62f7